engine/src/test/java/org/destinationsol/game/ShardTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.Body;
+import org.destinationsol.assets.Assets;
+import org.destinationsol.game.drawables.Drawable;
+import org.destinationsol.game.drawables.RectSprite;
+import org.destinationsol.testingUtilities.BodyUtilities;
+import org.destinationsol.testingUtilities.InitializationUtilities;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class ShardTest {
+
+    private static ArrayList<Drawable> drawables;
+
+    private static void init() {
+        InitializationUtilities.init();
+    }
+
+    private static final SolObject constantShard;
```

Comment:
I thought this looked out of order, but I had to realize the Checkstyle config file was bad before I could confirm that. Although this is an interesting line of static stuffs :-)

This line should be up next to the other plain variable, then the methods get to go next

To fix Checkstyle (I'll submit a separate PR): change `codeMetrics group: 'org.terasology.config', name: 'codemetrics', version: '1.1.0', ext: 'zip'` to use `1.2.0` in the project root `build.gradle` then rerun `gradlew idea. Then pick to use the "Terasology Checkstyle" and run a scan against this file to confirm.

At that point this variable name will also register a style warning since it is a constant, so it should be in all caps :-)

Any point to not just have `InitializationUtilities.init();` inside the static init so the plain `init` method can simply be removed?